devine/core/tracks/chapter.py

```diff
@@ -48,6 +49,13 @@ def named(self) -> bool:
         """Check if Chapter is named."""
         return bool(self.title)
 
+    @staticmethod
+    def time_to(sconds: str) -> str:
+        """Get time format from millisecond."""
+        time = datetime.utcfromtimestamp(sconds)
+        mils = ".000" if str(time.microsecond)[:3] == "0" else f".{str(time.microsecond)[:3]}"
```

Comment:
Can't you just change this to `mils = f".{str(time.microsecond)[:3]}"`? `str(...)[:3]` has no world of possibility to return `"0"` anyway, so it's likely already doing that else expression all the time anyway.

Reply:
One issue I see potentially happening is if the microseconds aren't 3 chars in length like mkvmerge expects them to be.

Reply:
If I use this ```mils = f".{str(time.microsecond)[:3]}"``` insted of this ```mils = ".000" if this str(time.microsecond)[:3] == "0" else f".{str(time.microsecond)[:3]}"```, in case of input like this `140.0 sconds`  that will happen ```00:02:20.0```.